kani-driver/src/args.rs

```diff
@@ -226,6 +230,15 @@ impl KaniArgs {
             Some(DEFAULT_OBJECT_BITS)
         }
     }
+
+    /// Computes how many threads should be used to verify harnesses.
+    pub fn jobs(&self) -> Option<usize> {
+        match self.jobs {
+            None => Some(1),          // no argument, default 1
+            Some(None) => None,       // -j
+            Some(Some(x)) => Some(x), // -j=x
```

Comment:
Isn't it worth to create an enum for all these options? Something like:
```
enum JobParalellism {
    Single,
    Multiple(Option<u32>),
}
```

Reply:
The double option in the args is a bit confusing imo, but is idiomatic there.

The option it returns exactly corresponds to calling `num_threads` on the thread pool (none means no, some means yes with this value), so I thought that was clear.